### DISCUSSION TITLE
Segmentation Fault

### DISCUSSION THREAD

**Discussion by jtjones3** [docksal/docksal#1773] [Q&A]
All of a sudden, when trying to run `fin drush uli`(or any `fin drush` command) on one repo I get a `segmentation fault` in the Terminal app. All the other repos (15+) are working fine. I'm at a loss as to why this is happening suddenly in one repo.

What have I tried? Delete the containers, local repo and Docker Desktop. Then restarted and reinstalled deleted entities and got the same error.

MacBook Pro (intel)
macOS 13.3.1
Docker Desktop 4.18.0
Docksal 1.18.2
Drupal 9.4.12
PHP 8.1
mariaDB 10.4

**Crash Report from Console application**
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               com.docker.vpnkit [1537]
Path:                  /Applications/Docker.app/Contents/Resources/bin/com.docker.vpnkit
Identifier:            com.docker.vpnkit
Version:               ???
Code Type:             X86-64 (Native)
Parent Process:        com.docker.backend [1398]
Responsible:           Docker [1395]
User ID:               1845524569

Date/Time:             2023-04-17 11:38:33.2060 -0700
OS Version:            macOS 13.3.1 (22E261)
Report Version:        12
Bridge OS Version:     7.4 (20P4252)
Anonymous UUID:        37C058A7-B013-0000-873B-217D0181E7A3

Time Awake Since Boot: 490 seconds

System Integrity Protection: enabled

Crashed Thread:        3

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x00007f8b10000000
Exception Codes:       0x0000000000000001, 0x00007f8b10000000

Termination Reason:    Namespace SIGNAL, Code 11 Segmentation fault: 11
Terminating Process:   exc handler [1537]

VM Region Info: 0x7f8b10000000 is not in any region.  Bytes after previous region: 1  Bytes before following region: 40894464
      REGION TYPE                    START - END         [ VSIZE] PRT/MAX SHRMOD  REGION DETAIL
      MALLOC_MEDIUM            7f8b0f800000-7f8b10000000 [ 8192K] rw-/rwx SM=PRV  
--->  GAP OF 0x2700000 BYTES
      MALLOC_TINY              7f8b12700000-7f8b12800000 [ 1024K] rw-/rwx SM=PRV  

Thread 0::  Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	    0x7ff8148dd282 __select + 10
1   com.docker.vpnkit             	       0x10de3be7e unix_select + 814

**Comment by lmakarov** [comment]
Since it's the `com.docker.vpnkit` process that's crashing on your Mac, the issue is upstream in Docker Desktop for Mac.
It's going to be very difficult to pinpoint the issue without a reliable way to reproduce it.
Is there something special about this project/repo compared to the rest of them (that do not crash Docker)?

**Comment by jtjones3** [comment]
Well yes. 3 years ago I inherited the site that crashes from another company so I'm sure there are some differences in the code that I haven't figured out yet. I think this problem first started happening after updating to Ventura a couple of weeks ago.

Posting on Docker Desktop support next.
